PopulationDensity.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
<p>You'll probably want to add <code>compressed/popdensity.zip</code>  to gitignore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>